src/components/Gouvernance/FeuilleDeRoute/DetailsFeuilleDeRoute.tsx

```diff
@@ -0,0 +1,107 @@
+import { ReactElement } from 'react'
+
+import Tag from '@/components/shared/Tag/Tag'
+import { GouvernanceViewModel } from '@/presenters/gouvernancePresenter'
+
+type FeuilleDeRoute = NonNullable<GouvernanceViewModel['sectionFeuillesDeRoute']['feuillesDeRoute']>[number];
+interface DetailsFeuilleDeRouteProps { readonly feuilleDeRoute: FeuilleDeRoute }
+
+export default function DetailsFeuilleDeRoute({ feuilleDeRoute }:DetailsFeuilleDeRouteProps):ReactElement {
+  return (
+    <>
+      <h2
+        className="color-blue-france fr-mt-5w"
+      >
+        {feuilleDeRoute.nom}
+      </h2>
+      <div
+        className="fr-mb-2w"
+      >
+        <div className="color-grey">
+          Responsable de la feuille de route
+        </div>
+        <Tag >
+          CC des Monts du Lyonnais
+        </Tag>
+      </div>
+      <div
+        className="fr-mb-2w"
+      >
+        <div className="color-grey">
+          Budget total des actions
+        </div>
+        <div className="font-weight-700">
+          {`${feuilleDeRoute.budgetGlobal}€`}
```

Comment:
```suggestion
          {`${feuilleDeRoute.budgetGlobal} €`}
```
Pareil pour les autres montants.
J'ai mis à jour la maquette sur ton ticket pour avoir la bonne version.